### ISSUE TITLE
Band overlap

### ISSUE THREAD

**Issue by G8JNJ** [jks-prv/Beagle_SDR_GPS#52]
Hi,

Just noticed there is an overlap of band markers between 3900 and 4000KHz.

The coloured bar is purple (mix of red and blue) and the titles obscure each other.

Regards,

Martin

**Comment by jks-prv** [comment]
Yes, there is an entry on the bug list that says: "Handle overlapping band segments better".

**Comment by G8JNJ** [comment]
OK, Sorry Hadn't spotted that one.

Regards,

Martin
